src/pages/publication/references_tab/JournalPublicationReferenceForm.tsx

```diff
@@ -1,12 +1,88 @@
 import React from 'react';
-import { Field } from 'formik';
+import { Field, useFormikContext } from 'formik';
 import { TextField, Select } from 'formik-material-ui';
-import { MenuItem } from '@material-ui/core';
+import { MenuItem, Radio, FormControl, FormLabel, FormControlLabel, RadioGroup } from '@material-ui/core';
 import { useTranslation } from 'react-i18next';
-import { journalPublicationTypes, journalPublicationFieldNames } from '../../../types/references.types';
+import { journalPublicationTypes } from '../../../types/references.types';
+import Journal from './Journal';
+import styled from 'styled-components';
+import PublicationChannelSearch from './PublicationChannelSearch';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import CancelIcon from '@material-ui/icons/Cancel';
+import InfoIcon from '@material-ui/icons/Info';
+
+const StyledArticleDetail = styled.div`
+  display: grid;
+  grid-template-columns: auto auto auto auto auto auto;
+  grid-column-gap: 1rem;
+  align-content: center;
+`;
+
+const StyledNVIValidation = styled.div`
```

Comment:
`StyledNviValidation`? (samme under)

Reply:
egentlig holder det kanskje med `StyledValidation` etc?

Reply:
Tenker at siden det er en validering for NVI at vi bør ha det med i navnet på komponenten